src/contexts/videoLibContext.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { createContext, useContext, useReducer, useState } from "react";
+
+import { vidLibReducer } from "../reducers/videoLibReducer";
+

```

Comment:
Use full naming instead of shorthand.

Reply:
Definitely, That's a valid change to make